modules/test/playwright/pages/FDS/FDS.page.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect} from '@playwright/test';
+
+import {ApplicationsMenuPage} from '../product-navigation-applications-menu/applicationsMenu.page';
+
+import type {Page} from '@playwright/test';
+
+export class FDSPage {
+	page: Page;
+	applicationsMenuPage: ApplicationsMenuPage;
+
+	constructor(page: Page) {
+		this.page = page;
+		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+	}
+
+	async goto() {
+		await this.applicationsMenuPage.goToFDS();
+	}
+
+	async createTestDataSet() {
```

Comment:
I find useful parametrizing all these DataSet creation and navigation methods, in the same way it is done with actions. 
Otherwise, we will end up with tons of `Data Set Test` entities but [accessing always to the first one](https://github.com/liferay-frontend/liferay-portal/pull/3924/files#diff-a94b577a1ae016e3d99d36b1e7430548338a6d07366f75ab74e9020e4b4c45b5R49)

Reply:
I think parametrizing DataSet test data isn't a good strategy. I prefer to clean up the data test stuff.

Same https://github.com/liferay-frontend/liferay-portal/pull/3924#discussion_r1449089489

Reply:
> I think parametrizing DataSet test data isn't a good strategy. I prefer to clean up the data test stuff.

Why? We need reasons for and against both models to decide which one is better in each case...

Reply:
I think test data to fill names, inputs and stuff, should be provided by the test, not hardcoded, since user inputs is another subject to test. In order to do that I know [this library](https://fakerjs.dev/), which provides fake but realistic data about many subjects. The advantage is that it uses a huge range of lenghts, simbols, as a human QA could do, so it provides a good coverage of cases. The downside, however, is the fact a test could fail for data that we don't even know